packages/swap/src/event-handlers/swapEgressScheduled.ts

```diff
@@ -107,18 +107,25 @@ export default async function swapEgressScheduled({
   // TODO: use an accurate source for determining the egress fee once the protocol provides one
   const egressFee = await getEgressFeeAtBlock(block.hash, swap.destAsset);
 
-  await prisma.swap.update({
-    where: { nativeId: swapId },
-    data: {
-      egressAmount: swap.destAmount?.sub(egressFee.toString()),
```

Comment:
this is what happens if you have two columns for the same thing, you forget to update one